make/bundle-dist

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+set -o xtrace
+set -o errexit
+set -o nounset
+
+GIT_ROOT="${GIT_ROOT:-$(git rev-parse --show-toplevel)}"
+
+. "${GIT_ROOT}/make/include/versioning"
+
+for OS in linux darwin
+do
+    tmp_dir=$(mktemp -d)
+    trap "rm -rf '${tmp_dir}'" EXIT
```

Comment:
There's only one `EXIT` trap, so this actually means we never delete the first `tmp_dir` on success.